apps/dashboard/src/utils/extractApiErrorMessage.ts

```
@@ -0,0 +1,16 @@
+export function extractApiErrorMessage(error: unknown): string | null {
+  if (
+    typeof error === 'object' &&
```

ESLint

[error] Replace `'object'` with `"object"`

(prettier/prettier)